vendordeps/AdvantageKit.json

```diff
@@ -3,7 +3,7 @@
   "name": "AdvantageKit",
   "version": "3.0.0-beta-6",
   "uuid": "d820cc26-74e3-11ec-90d6-0242ac120003",
-  "frcYear": "2024",
+  "frcYear": 2024,
```

Comment:
Shouldn't be changed in this branch. You can cherry-pick this change in a new branch (provided this change was introduced in its own commit, without other changes). Then, use interactive rebasing to remove this diff from this branch.

Reply:
e953530fddc56a6206032d8913572b394055b9a4